dask/dataframe/io/tests/test_io.py

```diff
@@ -969,3 +969,13 @@ def __call__(self, t):
     assert_eq(ddf["A"], expect["A"])
     assert set(projected) == {"A"}
     assert_eq(ddf, expect)
+
+
+@pytest.mark.gpu
+def test_dataframe_from_dict():
+    cudf = pytest.importorskip("cudf")
+    dask_cudf = pytest.importorskip("dask_cudf")
+    data = {"a": cudf.Series([1, 2, 3, 4]), "B": cudf.Series([10, 11, 12, 13])}
+    expected = cudf.DataFrame(data)
+    actual = dask_cudf.DataFrame.from_dict(data, npartitions=2)
+    assert_eq(expected, actual.compute())
```

Comment:
```suggestion
@pytest.mark.gpu
@pytest.mark.parametrize("backend", ["pandas", "cudf"])
def test_from_dict_backends(backend):
    _lib = pytest.importorskip(backend)
    with config.set({"dataframe.backend": backend}):
        data = {"a": [1, 2, 3, 4], "B": [10, 11, 12, 13]}
        expected = _lib.DataFrame(data)

        # Check dd.from_dict API
        got = dd.from_dict(data, npartitions=2)
        assert_eq(expected, got)
        
        # Check from_dict classmethod
        got_classmethod = got.from_dict(data, npartitions=2)
        assert_eq(expected, got_classmethod)
```

Maybe test backend dispatching more explicitly here?